controllers/suite_test.go

```diff
@@ -91,13 +55,21 @@ func setup() error {
 	// Uncomment the below to run the tests in the old v1 naming mode
 	// envy.Set("AZURE_SECRET_NAMING_VERSION", "1")
 
-	err := resourcemanagerconfig.ParseEnvironment()
+	err := config.ParseEnvironment()
 	if err != nil {
 		return err
 	}
 
+	if len(envy.Get("TEST_EMIT_ASO_LOGS", "")) > 0 {
+		ctrl.SetLogger(zap.New(func(o *zap.Options) {
```

Comment:
Any reason not to just do this all the time?

Reply:
It's really noisy - it would make the logs huge in the full test run, especially with them all running in parallel.